app/models/event.py

```diff
@@ -67,10 +66,12 @@ class Event(db.Model):
     attendees = db.relationship('TicketHolder', backref="event")
     privacy = db.Column(db.String, default="public")
     state = db.Column(db.String, default="Draft")
-    event_type_id = db.Column(db.Integer, db.ForeignKey('event_types.id', ondelete='CASCADE'))
-    event_topic_id = db.Column(db.Integer, db.ForeignKey('event_topics.id', ondelete='CASCADE'))
-    event_sub_topic_id = db.Column(db.Integer, db.ForeignKey(
-        'event_sub_topics.id', ondelete='CASCADE'))
+    event_type_id = db.Column(
+        db.Integer, db.ForeignKey('event_types.id', ondelete='CASCADE'))
+    event_topic_id = db.Column(
+        db.Integer, db.ForeignKey('event_topics.id', ondelete='CASCADE'))
+    event_sub_topic_id = db.Column(
+        db.Integer, db.ForeignKey('event_sub_topics.id', ondelete='CASCADE'))
```

Comment:
I think we should avoid making unrelated changes specially formatting ones in the PR. This not only makes reviewing complicated but also destroys git blame, in other words puts the blame on you. :wink: